packages/theme/getters/categoryGetters.ts

```diff
@@ -1,6 +1,5 @@
 import { CategoryGetters, AgnosticCategoryTree, AgnosticBreadcrumb } from '@vue-storefront/core';
-import { Category } from '@vue-storefront/magento-api';
-import { buildCategoryTree } from '../helpers/buildCategoryTree';
+import { Category, buildCategoryTree } from '@vue-storefront/magento';
```

Comment:
```suggestion
import { Category } from '@vue-storefront/magento';
```
We already have our own buildCategoryTree instance within the `theme` package. Can you please use it instead? They're duplicated for now but I think the package version will be removed soon